### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by rgbkrk**
Went on a spree adhering commutable to our eslint setup that we use elsewhere.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -67,10 +67,7 @@ export function insertCellAfter(notebook, cell, cellID, priorCellID) {
   return insertCellAt(notebook, cell, cellID, notebook.get('cellOrder').indexOf(priorCellID) + 1);
 }
 
-export function appendCell(notebook, cell, cellID) {
-  if(!cellID) {
-    cellID = uuid();
-  }
```
Comment (jdfreder) [inline src/index.js]: Did you make sure we aren't using this signature somewhere first?  Otherwise we'll need to make sure to bump a major.

Comment (jdfreder) [inline src/index.js]: I think I may have used it recently in the round trip tests in nteract/nteract

File: src/index.js [hunk 2]
```diff
@@ -67,10 +67,7 @@ export function insertCellAfter(notebook, cell, cellID, priorCellID) {
...
+export function appendCell(notebook, cell, cellID = uuid()) {
```
Comment (rgbkrk) [inline src/index.js]: @jdfreder ^^ The function is still here, I added a default argument.

On second thought, I wonder if this sets the uuid once or every time...

Comment (jdfreder) [inline src/index.js]: Seems like the spec would have it only generate one uuid.  But I could see the transpiled code behaving differently.  Good question!

Comment (rgbkrk) [inline src/index.js]: Yup, looks like the transpiled code will actually call out. We don't have to worry about the Python-like behavior:

```js
function appendCell(notebook, cell) {
  var cellID = arguments.length <= 2 || arguments[2] === undefined ? (0, _uuid.v4)() : arguments[2];

  return notebook.setIn(['cellMap', cellID], cell).set('cellOrder', notebook.get('cellOrder').push(cellID));
}
```

**Pull request merged**